R/derive_params_growth_age.R

```diff
@@ -97,10 +97,10 @@
 #' @export
 #'
 #' @examples
-#' library(dplyr)
-#' library(lubridate)
-#' library(rlang)
-#' library(admiral)
+#' suppressPackageStartupMessages(library(dplyr))
+#' suppressPackageStartupMessages(library(lubridate))
+#' suppressPackageStartupMessages(library(rlang))
+#' suppressPackageStartupMessages(library(admiral))
```

Comment:
@santarabantoosoo i checked a few admiral core functions and there they seem to use `library(dplyr, warn.conflicts = FALSE)` - can you double check this approach works ok and if so match it here.

I would add same to https://github.com/pharmaverse/admiralpeds/blob/main/R/derive_params_growth_height.R for consistency too.

Also feel free to add yourself to the DESCRIPTION file as a contributor as part of this PR.

Reply:
Thanks Ross. 

 I added warn.conflicts = FALSE to both files and added my name as a contributor.